_field-types/supported-field-types/star-tree.md

```diff
@@ -109,13 +109,8 @@ PUT logs
 
 ## Star-tree mapping parameters
 
-Specify any star-tree configuration mapping options in the `config` section. Parameters cannot be modified without reindexing documents.
+Specify any star-tree configuration mapping options in the `mappings` section. Parameters cannot be modified without reindexing documents.
```

Comment:
Should we differentiate from mappings since its present a bit deeper in config section.

under corresponding STIX `config` section under `mappings`

 or something similar ?

Reply:
For sure, let me rework the order of this section.